doc/modules/mod_event_pusher_http.md

```diff
@@ -0,0 +1,59 @@
+## Module description
+
+This module is a backend of [mod_event_pusher] that enables forwarding certain events (messages, presence, etc.) via HTTP to external services such as push (by mobile, email or SMS), big data, or analytics services.
+
+### How it works
+
+The module hooks on all packets sent by connected users.
+When the hook is triggered, the module:
+
+* runs a callback module's `should_make_req/3` function to see if a notification should be sent
+* sends a POST request composed of `{Host::binary(), Sender::binary(), Receiver::binary(), Message::binary()}` to the http notification server
+
+### Callback module
+
+To find out what should be sent to the HTTP server, MongooseIM calls `Mod:should_make_req(Packet::xmlel(), From::jid(), To::jid())`.
+By default it uses the function in `mod_event_pusher_http` itself, which ships all non-empty chat messages and nothing else. The module can be substituted here, the method should return `true | false`.
+
+## Prerequisites
+
+This module uses a connection pool created by mongoose_http_client. It must be defined in `http_connections` settings.
```

Comment:
This module uses a connection pool created by mongoose_http_client. It must be defined in the `http_connections` settings.